calculatorBackend/test/calculator_test[1]_tests.cmake

```diff
@@ -0,0 +1,35 @@
+add_test( whenTestingCalculator.WhenAddingPositiveAndNegativeIntegers_ThenCorrectValueReturned /mnt/c/Users/jared.vandewater/Learning/CalculatorSkeleton/calculatorBackend/test/calculator_test [==[--gtest_filter=whenTestingCalculator.WhenAddingPositiveAndNegativeIntegers_ThenCorrectValueReturned]==] --gtest_also_run_disabled_tests)
+set_tests_properties( whenTestingCalculator.WhenAddingPositiveAndNegativeIntegers_ThenCorrectValueReturned PROPERTIES WORKING_DIRECTORY /mnt/c/Users/jared.vandewater/Learning/CalculatorSkeleton/calculatorBackend/test SKIP_REGULAR_EXPRESSION [==[\[  SKIPPED \]]==])
```

Comment:
I don't believe these calculator_test[1] .cmake files are needed. I had a hard time removing with the gitignore, but you can manually delete them and when you stage and commit, manually disgard changes on these files